content-src/components/DiscoveryStreamComponents/DarkModeMessage/DarkModeMessage.jsx

```diff
@@ -0,0 +1,40 @@
+import {actionCreators as ac} from "common/Actions.jsm";
+import {connect} from "react-redux";
+import React from "react";
+
+export class _DarkModeMessage extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.handleSwitch = this.handleSwitch.bind(this);
+    this.handleCancel = this.handleCancel.bind(this);
+  }
+
+  handleSwitch() {
+    // TBD: Call switch to older version
+    // this.props.dispatch(ac.UserEvent({event: at.SWITCH_DEFAULT}));
```

Comment:
This will want to be `this.store.dispatch(ac.BroadcastToContent({type: at.DISCOVERY_STREAM_OPT_OUT}));`

Reply:
perfect, thanks!